app/helpers/validate/teachers.spec.js

```diff
@@ -0,0 +1,59 @@
+const assert = require('assert')
+
+const teachers = require('./teachers')
+
+describe('teachers', function () {
+  it('should return empty list when has not errors', function () {
+    const disciplinas = [
+        {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 6 spaces but found 8.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555382)
```suggestion
      {
```